starter-code/src/functions-and-arrays.js

```diff
@@ -1,6 +1,16 @@
 // Find the maximum
+function maxOfTwoNumbers(num1, num2) {
+  if (num1>num2) {
+    return num1;
+  } else if (num2 > num1) {
+    return num2;
```

Comment:
Is this moment, mean that `num2` is greater or equal than `num1` so we can just put `return num2`